collections/_article/dev-snapshot-godot-4-2-dev-4.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 4"
+excerpt: "Progressing steadily towards the beta stage, the fourth dev snapshot brings a lot of improvements all around."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-4.webp
+image_caption_title: "AssetPlacer"
+image_caption_description: "A plugin by CookieBadger"
+date: 2023-09-05 16:00:00
+---
+
+We're making great progress in the development branch for Godot 4.2! The faster paced release cycle we started following after the 4.0 release seems to be working well, with pull requests being opened, reviewed, and merged at a steady state. This fourth dev snapshot contains nearly 250 PRs merged in a little over 3 weeks since the [dev 3 snapshot](/article/dev-snapshot-godot-4-2-dev-3/).
```

Comment:
I *think* this is what you meant.

```suggestion
We're making great progress in the development branch for Godot 4.2! The faster paced release cycle we started following after the 4.0 release seems to be working well, with pull requests being opened, reviewed, and merged at a steady rate. This fourth dev snapshot contains nearly 250 PRs merged in a little over 3 weeks since the [dev 3 snapshot](/article/dev-snapshot-godot-4-2-dev-3/).
```